packages/cardpay-cli/scheduled-payment/execute.ts

```diff
@@ -0,0 +1,137 @@
+import { Argv } from 'yargs';
+import { getSDK } from '@cardstack/cardpay-sdk';
+import { getEthereumClients, getConnectionType, NETWORK_OPTION_ANY } from '../utils';
+import { Arguments, CommandModule } from 'yargs';
+import { getConstant } from '@cardstack/cardpay-sdk';
+
+export default {
+  command:
+    'execute <moduleAddress> <tokenAddress> <amount> <payeeAddress> <fixedUSDFee> <percentageFee> <executionGas> <maxGasPrice> <gasTokenAddress> <salt> <payAt> <gasPrice>',
+  describe: 'Execute a scheduled payment',
+  builder(yargs: Argv) {
+    return yargs
+      .positional('moduleAddress', {
+        type: 'string',
+        description: 'The address of scheduled payment module',
+      })
+      .positional('tokenAddress', {
+        type: 'string',
+        description: 'The address of the token being transferred',
+      })
+      .positional('amount', {
+        type: 'string',
+        description: 'Amount of tokens you would like transferred (in the smallest units of token)',
+      })
+      .positional('payeeAddress', {
+        type: 'string',
+        description: 'The address of the payee of scheduled payment',
+      })
+      .positional('fixedUSDFee', {
+        type: 'number',
+        description: 'Fixed USD fee (e.g. 0.25)',
+      })
+      .positional('percentageFee', {
+        type: 'number',
+        description: 'Percentage fee (e.g. 5%, 0.05)',
+      })
+      .positional('percentageFee', {
+        type: 'number',
+        description: 'Percentage fee (e.g. 5%, 0.05)',
+      })
+      .positional('executionGas', {
+        type: 'string',
+        description: 'The gas limit to execute scheduled payment',
+      })
+      .positional('maxGasPrice', {
+        type: 'string',
+        description: 'Maximum gas price (in the smallest units of gas token)',
+      })
+      .positional('gasTokenAddress', {
+        type: 'string',
+        description: 'The address of the gas token',
+      })
+      .positional('salt', {
+        type: 'string',
+        description: 'Arbitrary string to make SP unique',
+      })
+      .positional('payAt', {
+        type: 'number',
+        description: 'Time to execute scheduled payments (in seconds)',
```

Comment:
We can copy this description from the other PR where we described this as UTC unix time

Reply:
copied